_aggregations/pipeline/derivative.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: default
+title: Derivative
+parent: Pipeline aggregations
+nav_order: 70
+redirect_from:
+  - /query-dsl/aggregations/pipeline-agg#derivative/
+  - /query-dsl/aggregations/pipeline/derivative/
+---
+
+# Derivative aggregations
+
+The `derivative` aggregation is a parent aggregation used to calculates first-order and second-order derivatives of each bucket of an aggregation. "First-order derivative" and "second-order derivative" are often shortened to "first derivative" and "second derivative" respectively. This page uses the shortened terms.
+
+In mathematics, the first derivative of a function is its instantaneous rate of change at any point in its domain. This is commonly represented as the slope of a tangent on a graph of the function. To learn more about derivatives, see [Wikipedia](https://en.wikipedia.org/wiki/Derivative).
+
+For an ordered series of buckets, `derivative` approximates a first derivative as the difference between metric values in the current previous buckets.
+
+The second derivative indicates how the rate of change of a quantity is itself changing, approximated as the difference between the current and previous first derivatives.
+
+A derivative can be positive or negative, indicating an increasing or decreasing function respectively.
+
+## Parameters
+
+The `derivative` aggregation takes the following parameters.
+
+| Parameter             | Required/Optional | Data type       | Description |
+| :--                   | :--               |  :--            | :--         |
+| `buckets_path`        | Required          | String          | The path of the aggregation buckets to be aggregated. See [Pipeline aggregations]({{site.url}}{{site.baseurl}}/aggregations/pipeline/index#buckets-path-property). |
+| `gap_policy`          | Optional          | String          | The policy to apply to missing data. Valid values are `skip` and `insert_zeros`. Default is `skip`. |
```

Comment:
Maybe rename the "gaps" section "gap policy" in the index page and link to that here?

Reply:
```suggestion
| `gap_policy`          | Optional          | String          | The policy to apply to missing data. Valid values are `skip` and `insert_zeros`. Default is `skip`. See [Data gaps]({{site.url}}{{site.baseurl}}/opensearch/aggregations/pipeline/index/#data-gaps). |
```